cszczepaniak/cribbly-frontend
# Add auth0

# What I added (link any issues addressed)
I added a simple and ugly interface to the backend. It features a login and logout button as well as two buttons to access the public and private endpoints on the backend.

# How to test it
1. Run the backend locally
2. Run the frontend with `npm start`
3. Navigate to `localhost:3000` in a browser. Try the buttons. Clicking the private button should give you an error when you're not logged in and return a response when you are logged in. Clicking the public button should always work.
